test/boost/comparable_bytes_test.cc

```diff
@@ -204,3 +204,17 @@ BOOST_AUTO_TEST_CASE(test_float) {
 BOOST_AUTO_TEST_CASE(test_double) {
     byte_comparable_test(floating_point_test_data_generator<double>());
 }
+
+extern void encode_varint_length(uint64_t length, uint8_t sign_only_byte, bytes_ostream& out);
+extern uint64_t decode_varint_length(managed_bytes_view& src, uint8_t sign_only_byte);
+BOOST_AUTO_TEST_CASE(test_varint_length_encoding) {
+    for (int shift = 0; shift <= 64; shift++) {
+        uint64_t length = (uint64_t(1) << shift) - 1;
+        for (uint8_t sign_only_byte : {0, 0xFF}) {
+            bytes_ostream out;
+            encode_varint_length(length, sign_only_byte, out);
```

Comment:
How is the test using this method, when it is only defined in the `.cc` file, I see no declaration in a header.